Makefile

```diff
@@ -139,12 +139,11 @@ bindings: .create-bindings .strip-autogen-comment .strip-nvml-h-linenumber
 .create-bindings: $(PKG_BINDINGS_DIR)/nvml.h $(SOURCES) | $(PKG_BINDINGS_DIR)
 	cp $(GEN_BINDINGS_DIR)/nvml.yml $(PKG_BINDINGS_DIR)
 	c-for-go -out $(PKG_DIR) $(PKG_BINDINGS_DIR)/nvml.yml
```

Comment:
Question: Instead of copying the `yml` file in the previous step, could we specify `$(GEN_BINDINGS_DIR)/nvml.yml` as input here? Then we also wouldn't need to remove it later?

Reply:
When I do that I get an incorrect `pkg/types_gen.so` file that gets generated. It's unclear why this is the case, but the fact that it doesn't "just work" when simplifying this means that it likely warrants a new PR to investigate.